src/main/java/com/fortanix/sdkms/performance/sampler/AbstractSignatureSampler.java

```diff
@@ -54,14 +58,7 @@ public void setupTest(JavaSamplerContext context) {
                 throw new ProviderException(e.getMessage());
             }
         }
-        MessageDigest md = null;
```

Comment:
Any reason to remove this code? This is done so that we don't send large input text to the API repeatedly. We generate hash once and send that to Sign API instead.

Reply:
it is using JCE crypto algorithm which has different semantic than SDKMS api. For example, JCE expects digest algorithm "SHA-256" but SDKMS API expects SHA256.
I have restored the changes but now we can only use SHA1, SHA-256, SHA-384, SHA512 algorithms.

Reply:
I don't think we need to use anything other than SDKMS symantics. Lets take SDKMS enums as input directly, rather than defining a new level of mapping.

Reply:
Fixed. Now we are using sdkms-java-sdk for digest compute